.github/workflows/ci.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
       fail-fast: true
       matrix:
         os: ["ubuntu-latest"]
-        python-version: ["3.7", "3.8"]
+        python-version: ["3.8", "3.9"]
```

Comment:
pygac still builds against 3.7 and 3.8, should we fix that too? also, what about 3.10?

Reply:
Good idea, done!